Neos.ContentRepository.Core/Classes/Projection/ContentGraph/NodeAggregate.php

```diff
@@ -87,6 +98,11 @@ public function getNodes(): iterable
         return array_values($this->nodesByOccupiedDimensionSpacePoint);
     }
 
+    public function getSingleNodeIndependentOfItsDimension(): Node
```

Comment:
alternative naming:

- `getPrimaryNode`
- `getArbitraryNode`
- `getOneNode`
- `getFirstNode`

Reply:
... or we could use the `$arbitraryDimensionSpacePoint` pattern to and pass it to`getNodeByCoveredDimensionSpacePoint`